src/commands/mod.rs

```diff
@@ -1413,12 +1413,16 @@ fn log_template(settings: &UserSettings) -> String {
     } else {
         "committer.timestamp()"
     };
+    let desired_id_len = settings
+        .log_id_preferred_length()
+        .map(|l| l.to_string())
+        .unwrap_or_default();
```

Comment:
Suppose we'll change the default `shortest*()` behavior, it's better to always specify the length parameter. It also removes the need of  `.to_string()`.

Reply:
Sure, might as well change that here.